### PR TITLE
Sam

### PR DESCRIPTION
**Pull request by SEStarkman**
updated sam.txt

### CODE REVIEW

File: sam.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Hi I am Sam!!
+My birthday is January 16, 1997
+I am going to be a sophomore at NEU next fall.
+I love puppies!
```
Comment (OatmealSoldier) [inline sam.txt]: what about cats?

Comment (kkimelman) [inline sam.txt]: what about me?

Comment (yiskylee) [inline sam.txt]: Me too!

File: sam.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Hi I am Sam!!
```
Comment (emersonboyd) [inline sam.txt]: Last name?

**Pull request merged**